pilot/pkg/bootstrap/namespacecontroller.go

```diff
@@ -39,27 +40,70 @@ const (
 	// can update its CA certificate in a ConfigMap in every namespace.
 	namespaceResyncPeriod = time.Second * 30
 	// The name of the ConfigMap in each namespace storing the root cert of non-Kube CA.
-	CACertNamespaceConfigMap      = "istio-ca-root-cert"
-	CACertNamespaceInsertInterval = time.Second
-	CACertNamespaceInsertTimeout  = time.Second * 2
+	CACertNamespaceConfigMap = "istio-ca-root-cert"
```

Comment:
Even not related, i still want to know why choose to store cert by a configmap not secret.

Reply:
The CA's cert (public key) isn't secret.